src/jdk.incubator.jextract/share/classes/jdk/internal/jextract/impl/TypedefBuilder.java

```diff
@@ -43,37 +41,6 @@ String superClass() {
 
     @Override
     JavaSourceBuilder classEnd() {
-        if (superClass == null) {
-            emitAllocatePointerMethods();
-        }
         return super.classEnd();
     }
-
-    void emitAllocatePointerMethods() {
-        // allocatePointer
-        incrAlign();
-        indent();
-        append(MEMBER_MODS);
-        append(" MemorySegment allocatePointer() {\n");
-        incrAlign();
-        indent();
-        append("return MemorySegment.allocateNative(C_POINTER);\n");
-        decrAlign();
-        indent();
-        append("}\n");
-        decrAlign();
-
-        // allocatePointer (scope version)
-        incrAlign();
-        indent();
-        append(MEMBER_MODS);
-        append(" MemorySegment allocatePointer(NativeScope scope) {\n");
-        incrAlign();
-        indent();
-        append("return scope.allocate(C_POINTER);\n");
-        decrAlign();
-        indent();
-        append("}\n");
-        decrAlign();
-    }
```

Comment:
I don't get why this is removed. Was it not being used?

Reply:
It was a leftover from when we were using the @C annotation. These methods don't do anything special other than allocating a C_POINTER

Reply:
Ok. (It doesn't look all that interesting either :) )